openproblems/tasks/cell_cell_communication/datasets/tnbc_wu2021.py

```diff
@@ -0,0 +1,14 @@
+from ....data.tnbc_wu2021 import load_tnbc_data
+from ....tools.decorators import dataset
+
+
+@dataset(
+    "~43k Cells from Triple Negative Breast Cancer",
+    data_url=load_tnbc_data.metadata["data_url"],
+    data_reference=load_tnbc_data.metadata["data_reference"],
+    dataset_summary="A single-cell atlas of human breast cancers with inferred"
+    " cytokine activities as assumed benchmark truth",
```

Comment:
I described both assumed truths, but since 280chars are a bit restrictive for details - I instead refer the user to the publication where we defined them.

Reply:
Thanks!